src/index.html

```diff
@@ -1,13 +1,71 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta 
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,   maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link 
+      rel="stylesheet"
+      href="./style.css"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@500&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div 
+      class="card"
+      data-qa="card" 
+    >
+      <section class="card__content">
+        <img 
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="Imac-picture"
+        >
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="card__text">
+          Product code: 195434
+        </p>
+
+        <div class="stars">
+          <div class="stars__container">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="stars__reviews">
+            Reviews: 5
+          </p>
+        </div>
```

Comment:
```suggestion
        <div class="card__reviews-container">
          <div class="stars stars--4">
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
            <div class="stars__star"></div>
          </div>
          <p class="card__reviews-text">
            Reviews: 5
          </p>
        </div>
```
Make it work like that